src/env/node.js

```diff
@@ -139,6 +147,7 @@ Press <b>^C</b> (<b>Ctrl+C</b>) or <b>q</b> to quit interactive mode.
 			readline.emitKeypressEvents(process.stdin);
 			process.stdin.setRawMode(true); // handle keypress events instead of Node
 
+			makeCollapsible(root, {...options, expand: options?.expand ?? "fail"});
```

Comment:
I think this behavior should be the default. Also, there are no docs, so I'm not sure how `expand` is meant to be used.